cmd/kanikoExecute_test.go

```diff
@@ -59,7 +65,8 @@ func TestRunKanikoExecute(t *testing.T) {
 			BuildSettingsInfo:           `{"mavenExecuteBuild":[{"dockerImage":"maven"}]}`,
 		}
 
-		runner := &mock.ExecMockRunner{}
```

Comment:
All instances of variable `runner ` is replaced to `execRunner` to differentiate it from new variable `shellRunner`